src/components/Modal/Modal.js

```diff
@@ -20,8 +20,19 @@ const Modal = () => {
                 </li>
             </ul>
             <div className="">Table of contents</div>
-            <ul className="pagination">
-
+            <ul className="modal-pagination">
```

Comment:
I guess it is a little awkward... this pagination is dynamic? I mean, when we have data for 4 or 5 pages, will it adapt? Or is this just a mock?